scripts/lib/aws_elasticache_testutil.sh

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env bash
+
+#################################################
+# functions for tests
+#################################################
+
+# print_k8s_ack_controller_pod_logs prints kubernetes ack controller pod logs
+# this function depends upon testutil.sh
+print_k8s_ack_controller_pod_logs() {
+  local ack_ctrl_pod_id=$( controller_pod_id )
+  kubectl logs -n ack-system "$ack_ctrl_pod_id"
+}
```

Comment:
++

We should pull this into `scripts/lib/k8s.sh` (some future PR, doesn't need to be this one)